debug/set_up_keyvault.sh

```diff
@@ -0,0 +1,136 @@
+# Check if User is authenticated with azure login
+
+if [ "$#" -eq 0 ]; then
+  echo "USAGE: ./set_up_keyvault.sh [\$RG_NAME] [\$DOCKER_USERNAME]"
+  exit 0
+fi
+
+# tr: Illegal byte sequence
+export LC_ALL=C
+export RG_NAME=$1
+export DOCKER_USER=$2
+export LOC=westus
```

Comment:
We should get the location from the user too? I think we can have the user set the following env vars:

- `RESOURCE_GROUP`
- `KEYVAULT_NAME`
- `AZURE_LOCATION`

Reply:
`export KEYVAULT_NAME=kv$RG_NAME-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)` generates a keyvault name based off of the RG name... would you think it's best for them to name it explicitly themselves?